osu.Game/Overlays/Direct/PanelDownloadButton.cs

```diff
@@ -66,7 +67,8 @@ private void load(OsuGame game, BeatmapManager beatmaps)
                         break;
 
                     default:
-                        beatmaps.Download(BeatmapSet.Value, noVideo);
+                        var minimiseDownloadSize = noVideo ?? osuConfig.GetBindable<bool>(OsuSetting.PreferNoVideo).Value;
```

Comment:
I'm not sure that the bindable should be fetched every time the button is clicked. Might be a little leaner by having a single local bound copy in the button itself and reading from that instead.

Reply:
Can you even press these buttons more than once? But sure, this should be changed, yes.

Reply:
Possibly, if the download fails.

Reply:
Not sure if I did it correctly. It seems that if you change the setting while in direct, the setting doesn't apply to old panel buttons (or maybe even this direct session?)